core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -397,12 +418,10 @@ contract Liquidatable is PricelessPositionManager {
         return liquidations[sponsor];
     }
 
-    /**
-     * @notice This settles a liquidation if it is in the PendingDispute state. If not, it will immediately return.
-     * If the liquidation is in the PendingDispute state, but a price is not available, this will revert.
-     * @param liquidationId to uniquely identify the dispute to settle
-     * @param sponsor the address of the sponsor who's dispute is being settled
-     */
```

Comment:
nit: even if we want to remove the natspec formatting, I think some of this comment remains useful to explain what the method does.

Reply:
ye, fair enough. have added it back in a diffrent format.